core/src/main/scala/kafka/admin/AclCommand.scala

```diff
@@ -61,8 +61,7 @@ object AclCommand {
   def withAuthorizer(opts: AclCommandOptions)(f: Authorizer => Unit) {
     var authorizerProperties = Map.empty[String, Any]
     if (opts.options.has(opts.authorizerPropertiesOpt)) {
-      val props = opts.options.valuesOf(opts.authorizerPropertiesOpt).asScala.map(_.split("="))
-      props.foreach(pair => authorizerProperties += (pair(0).trim -> pair(1).trim))
+      CommandLineUtils.parseKeyValueArgs(opts.options.valuesOf(opts.authorizerPropertiesOpt).asScala).asScala.foreach(x => authorizerProperties += (x._1 -> x._2))
```

Comment:
What about something like below to avoid var, and break up the long line?

```scala
val authorizerProperties = 
      if (opts.options.has(opts.authorizerPropertiesOpt)) {
        val properties = opts.options.valuesOf(opts.authorizerPropertiesOpt).asScala
        CommandLineUtils.parseKeyValueArgs(properties).asScala
      } else {
        Map.empty[String, Any]
      }
```